custom_components/atrea/climate.py

```diff
@@ -256,6 +260,17 @@ def fan_modes(self):
     def program(self):
         return self.air_handling_control
 
+    @property
+    def hvac_action(self):
+        if (self._heating == 1):
+            return HVACAction.HEATING
+        elif (self._cooling == 1):
+            return HVACAction.COOLING
+        elif (self._current_hvac_mode == HVAC_MODE_OFF):
+            return HVACAction.OFF
+        else:
+            return HVACAction.IDLE
```

Comment:
Wouldn't `return HVACAction.FAN` be better than `return HVACAction.IDLE`?

Reply:
You are right. Fan is better. Fixed in the latest commit. I was mainly after the cooling/heating as the history then shows the state nicely in the graph.